uxarray/grid.py

```diff
@@ -8,9 +8,10 @@
 from ._ugrid import _read_ugrid, _encode_ugrid
 from ._shapefile import _read_shpfile
 from ._scrip import _read_scrip, _encode_scrip
-from .helpers import get_all_face_area_from_coords, parse_grid_type, _convert_node_xyz_to_lonlat_rad, _convert_node_lonlat_rad_to_xyz
+from .helpers import get_all_face_area_from_coords, parse_grid_type, node_xyz_to_lonlat_rad, node_lonlat_rad_to_xyz
 
 int_dtype = np.uint32
+_FillValue = float("nan")
```

Comment:
This should be defined within the `Grid` class and it shouldn't be internal since being able to access `Grid.FillValue` makes sense outside of the scope of the class (unless we want to keep it as `_FillValue` to align closer with the UGRID conventions)

Include it in the `Grid` class and reference it as `self.FillValue`

Reply:
I am not very sure about the design of this implementation as well because of the following reasons:
1. We still haven't decided what's our universal `_Fill_Value`
2. What's the final dataset read-in and `Grid` initialization function
3. What does the `Grid` class look like finally

But I am happy to make any change to make the design look more reasonable

Reply:
Currently I just leave it as the `self.FillValue`

Reply:
Pinging @erogluorhan to see what his thoughts are about this.

Reply:
Resolved by #241